mtp_cashbook/assets-src/stylesheets/views/_disbursement-search.scss

```diff
@@ -28,7 +28,40 @@
     width: 100%;
   }
 
+  .mtp-description.mtp-description {
+    font-size: 13px;
+    font-weight: bold;
+  }
+
   .mtp-batch small {
     font-size: 16px;
   }
+
+  .mtp-sender {
+    width: 10%;
+  }
+
+  .mtp-amount {
+    width: 30%;
+  }
+
+  .mtp-recipient {
+    width: 15%;
+  }
+
+  .mtp-status {
+    width: 5%;
+    font-weight: bold;
+  }
+
+  .mtp-status--sent, .mtp-status--confirmed {
+    color: green;
```

Comment:
try `$green` or `$turquoise` instead or if they don't have the right contrast, try `$button-colour`